README.md

```diff
@@ -1,38 +1,91 @@
 
-Full reference documentation can be found in the [Nix manual](https://nix.dev/reference/nix-manual).
+## Installing Determinate
 
-## Building and developing
+You can install Determinate on [macOS](#macos), non-NixOS [Linux](#linux), and [NixOS](#nixos).
 
-Follow instructions in the Nix reference manual to [set up a development environment and build Nix from source](https://nix.dev/manual/nix/development/development/building.html).
+### macOS
 
-## Contributing
+On macOS, we recommend using the graphical installer from Determinate Systems.
+Click [here][gui] to download and run it.
+
+### Linux
+
+On Linux, including Windows Subsystem for Linux (WSL), we recommend installing Determinate Nix using [Determinate Nix Installer][installer]:
+
+```shell
+curl -fsSL https://install.determinate.systems/nix | sh -s -- install --determinate
+```
+
+### NixOS
+
+On [NixOS], we recommend following our [dedicated installation guide][nixos-install].
 
-Check the [contributing guide](./CONTRIBUTING.md) if you want to get involved with developing Nix.
+## Other resources
 
-## Additional resources
+Nix was created by [Eelco Dolstra][eelco] and developed as the subject of his 2006 PhD thesis, [The Purely Functional Software Deployment Model][thesis].
+Today, a worldwide developer community contributes to Nix and the ecosystem that has grown around it.
 
-Nix was created by Eelco Dolstra and developed as the subject of his PhD thesis [The Purely Functional Software Deployment Model](https://edolstra.github.io/pubs/phd-thesis.pdf), published 2006.
-Today, a world-wide developer community contributes to Nix and the ecosystem that has grown around it.
+- [Zero to Nix][z2n], Determinate Systems' guide to Nix and [flakes] for beginners
+- [Nixpkgs], a collection of well over 100,000 software packages that you can build and manage using Nix
+- [NixOS] is a Linux distribution that can be configured fully declaratively
+- The Nix, Nixpkgs, and NixOS community on [nixos.org][website]
 
-- [The Nix, Nixpkgs, NixOS Community on nixos.org](https://nixos.org/)
-- [Official documentation on nix.dev](https://nix.dev)
-- [Nixpkgs](https://github.com/NixOS/nixpkgs) is [the largest, most up-to-date free software repository in the world](https://repology.org/repositories/graphs)
-- [NixOS](https://github.com/NixOS/nixpkgs/tree/master/nixos) is a Linux distribution that can be configured fully declaratively
-- [Discourse](https://discourse.nixos.org/)
-- Matrix: [#users:nixos.org](https://matrix.to/#/#users:nixos.org) for user support and [#nix-dev:nixos.org](https://matrix.to/#/#nix-dev:nixos.org) for development
+## Reference
+
+You can find full reference documentation in the [Determinate Nix manual][manual].
+This resource is a work in progress.
```

Comment:
```suggestion
The primary documentation for Determinate and Determinate Nix is available at [docs.determinate.systems][determinate].
For deeply technical reference material, see the [Determinate Nix manual][manual] which is based on the upstream Nix manual.
```

How about something like this? Calling it a work in progress when it doesn't actually receive great focus isn't my favorite thing